src/views/tickets/tickets.jsx

```diff
@@ -120,85 +120,6 @@ export function Tickets(props) {
     setViewedTicket((prevState) => (prevState ? null : ticket));
   };
 
-  const columns = [{
```

Comment:
Looks like these and the ticketFormStructure got removed by accident!

Reply:
Hey @beekman ! I went ahead and replaced the missing components and fixed up a couple things so we can get this merged. I hope that's alright with you and you're doing alright!